.github/workflows/cicd.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["2.7", "3.6", "3.7", "3.8", "3.9"]
+        python-version: ["2.7", "3.6", "3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
looks like 3.6 is EOL, and we should consider removing support for this and 3.7 in next major

Reply:
https://vfxplatform.com/platform_history.html